src/inputValidationError.js

```diff
@@ -33,6 +33,10 @@ const buildMessage = function(error){
         return `${error.message} [${error.params.allowedValues.toString()}]`;
     }
 
+    if (error.keyword === 'additionalProperties') {
+        return `${error.message} [${error.params.additionalProperty.toString()}]`;
```

Comment:
IMO, the additional properties should be presented inside quotes and not in brackets, as there is only one property in each error message

Reply:
done